meta/main.yml

```diff
@@ -16,20 +16,27 @@ galaxy_info:
         - wheezy
         - jessie
         - stretch
+        - buster
     - name: EL
       versions:
+        - 8
         - 7
         - 6
     - name: opensuse
       versions:
-        - 11
-        - 12
+        - 12.1
+        - 12.2
+        - 12.3
+        - 13.1
+        - 13.2
```

Comment:
Should we also add the SLES versions we support (11SP3, 11SP4, 12, 12SP1, 15)?

Reply:
oh, nice find, didn't see `SLES` is regarded as a separate platform altogether. I'll update 👍